packages/ignition/src/app/structured-data/intangible.ts

```diff
@@ -0,0 +1,8 @@
+import { Thing } from ".";
```

Comment:
Do not use import like that, it creates redundant references. Next can handle that, so, it works, but it's not ok